chapter_natural-language-processing-pretraining/bert-pretraining.md

```diff
@@ -198,6 +196,7 @@ def train_bert(train_iter, net, loss, vocab_size, devices, num_steps):
             mlm_l, nsp_l, l = _get_batch_loss_bert(
                 net, loss, vocab_size, tokens_X, segments_X, valid_lens_x,
                 pred_positions_X, mlm_weights_X, mlm_Y, nsp_y)
+            #nn.DataParallel(net, device_ids=devices)
```

Comment:
Please remove this

Reply:
Thanks for pointing out. it is removed in the latest submission.